skimage/measure/simple_metrics.py

```diff
@@ -142,3 +145,50 @@ def compare_psnr(im_true, im_test, data_range=None):
 
     err = compare_mse(im_true, im_test)
     return 10 * np.log10((data_range ** 2) / err)
+
+
+def enhancement_measure(image: np.ndarray,
+                        size: int = 3) -> float:
+    """
+        The image enhancement measure called EME based on [1]_.
+
+        Parameters
+        ----------
+        image : array
+            Input image of which the quality should be assessed.
+            Can be either 3-channel RGB or 1-channel grayscale.
+            The function converts pixel intensities into floats by default.
+        size : int
+            Size of the window. Default value is 3.
+
+        Returns
+        -------
+        eme : float
+            The number describing image quality.
+
+        References
+        ----------
+        .. [1] Agaian, Sos S., Karen Panetta, and Artyom M. Grigoryan.
+               "A new measure of image enhancement."
+               IASTED International Conference on Signal Processing
+               & Communication. Citeseer, 2000.
+        Examples
+        --------
+        >>> from skimage.data import camera
+        >>> from skimage.exposure import equalize_hist
+        >>> img = camera()
+        >>> print("Image quality:\n")
+            Image quality:
+        >>> print(f"\tbefore: {quality(img)}")
+                before: 0.9096745071475523
+        >>> print(f"\tafter: {quality(equalize_hist(img))}")
+                after: 1.299327371881219
+
+    """
+    image = img_as_float(image)
+    eme = np.zeros_like(image)
+    footprint = tuple([size] * len(image.shape))
+    eme = np.divide(maximum_filter(image, footprint=footprint) + 1,
+                    minimum_filter(image, footprint=footprint) + 1)
```

Comment:
Is adding `+1` here supported by the linked paper? Why is this necessary? Do you want to avoid division by 0 this way?

Reply:
So, in the paper the authors don't add anything, but division by zero is a possibility here, so to avoid it I add +1 to both numerator and denominator. At least in my experiments I used the function in this form a lot. It might make sense to use something like an `eps = 1` as a parameter passed to the function. I will check which values make sense.
Should the implementation follow the same comparison pattern as with PSNE and other functions? `compare_enhancement` or something like that?

Reply:
Sorry, forgot to address the question about the image. I don't personally have it, but google search gave me [this link](https://link.springer.com/chapter/10.1007%2F978-3-642-23154-4_11) to the source paper and [the image](https://www.google.com/search?sa=G&hl=en&q=monochrome&tbm=isch&source=iu&ictx=1&tbs=simg:CAESvgIJt1bAc_1cJHhMasgILEKjU2AQaBAgUCAoMCxCwjKcIGmEKXwgDEieVAYESlwH6AYMS7AV_1qgbMAvsBiz7wJ6c27if2NaU29DbyJ6I2kz4aMBrLMEAUP2pMseBtDd5bfccLZkF_1HpcsD1s4kW6GfqcgdI3SV3g-nobzAkj7i1DbYSAEDAsQjq7-CBoKCggIARIE6a_1fCgwLEJ3twQkangEKHQoKbW9ub2Nocm9tZdqliPYDCwoJL20vMDFtd2tmCh4KC2FsYXJtIGNsb2Nr2qWI9gMLCgkvbS8wNDZkbHIKGgoHYW50aXF1ZdqliPYDCwoJL20vMDF5czJzCiIKD2JsYWNrLWFuZC13aGl0ZdqliPYDCwoJL20vMDFnNmdzCh0KCnN0aWxsIGxpZmXapYj2AwsKCS9tLzAxc3B6cww&fir=VQ-TGHm2qMMWYM%253A%252CvLAo2pnVeI90MM%252C_&vet=1&usg=AI4_-kQx9DbbxFxkhS3ywjL0w-seNQZmRQ&ved=2ahUKEwjR3435g_TgAhUJVN8KHRurCmoQ9QEwAnoECAAQBA#imgrc=VQ-TGHm2qMMWYM:). Should the image be added to tests here?

Reply:
I'm not sure that simply adding +1 is the right way to go. According to [this guide](http://scikit-image.org/docs/stable/user_guide/data_types.html) images of dtype float may also range from -1 to 1. If we pass in an image with a -1 somewhere adding +1 doesn't work.

No, adding the image to the tests shouldn't be necessary. If you want to use a natural image there you can use the [ones already shipped with scikit-image](http://scikit-image.org/docs/stable/api/skimage.data.html).